api/src/wfl/environment.clj

```diff
@@ -48,6 +48,8 @@
    #(-> nil)
    "WFL_FIRECLOUD_URL"
    #(-> "https://api.firecloud.org")
+   "WFL_RAWLS_URL"
+   #(-> "https://rawls.dsde-dev.broadinstitute.org")
```

Comment:
Per Ed's comment in Slack, we can only interact with TDR from Rawls when both are in the same environment, i.e. both dev or both prod.  The existing TDR in this file points to dev.

Reply:
Thanks for flagging it, makes sense to me. In order to use prod Terra and RAWLs in WFL, we probably need to update the ENVs in deployment helmfile.

Reply:
Good to know, that makes sense.